dev_docs/tutorials/debugging.mdx

```diff
@@ -61,3 +61,49 @@ logging:
     - name: elasticsearch.query
       level: debug
 ```
+
+## Debugging Kibana with APM
+
+Kibana is integrated with APM's node and RUM agents. 
+To learn more about how APM works and what it reports, refer to the [documentation](https://www.elastic.co/observability/solution-overview).
+
+We currently track the following transactions from Kibana: 
```

Comment:
Reading the [data model docs](https://www.elastic.co/guide/en/apm/guide/current/data-model-transactions.html) I presume that the list below is refering to "types of transactions" but not "transactions"?